manifests/init.pp

```diff
@@ -24,7 +24,7 @@
   }
 
   package { 'puppet-bolt':
-    ensure  => present,
+    ensure  => '3.26.2',
```

Comment:
Shall we make the version into a parameter defaulting to the currently pinned version?

Reply:
Hi @timidri usually, I would like the idea of parameterizing a variable, but in this case, I'm concerned that it will be a pitfall, it would be like releasing PE with the option of setting different PostgreSQL versions.